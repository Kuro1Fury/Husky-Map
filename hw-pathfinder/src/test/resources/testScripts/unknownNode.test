# Test unknown nodes

CreateGraph A

AddNode A n1

FindPath A n1 n2

FindPath A n2 n1

FindPath A n2 n3