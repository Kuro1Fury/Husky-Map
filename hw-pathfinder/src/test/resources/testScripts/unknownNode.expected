# Test unknown nodes

created graph A

added node n1 to A

unknown: n2

unknown: n2

unknown: n2
unknown: n3